.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.3.5
+current_version = 2.0.0a1
```

Comment:
General todo, remove any deprecated function referenced by `@deprecated`

Reply:
Preferably we do this in issue #342 which has been created for this purpose. We can still do that as part of the 2.0.0 milestone, just not in this monster PR :)